docs/pages/repo/docs/reference/command-line-reference.mdx

```diff
@@ -293,6 +293,16 @@ turbo run build --global-deps=".env.*" --global-deps=".eslintrc" --global-deps="
 
 You can also specify these in your `turbo` configuration as `globalDependencies` key.
 
+#### `--framework-inference`
+
+`type: bool`
+
+Specify whether or not to do framework inference for tasks. Defaults to `true`, can be configured to be `false` which skips framework inference for tasks. This disables [automatic environment variable inclusion](../core-concepts/caching#automatic-environment-variable-inclusion).
+
+```sh
+turbo run build --framework-inference=false
+```
```

Comment:
Typically want this in a separate PR with a [docs: next release](https://github.com/vercel/turbo/labels/docs%3A%20next%20release) label

Reply:
This would go out in the next patch so I elected short-term disagreement to reduce process overhead. The commit is separate, though, for exactly this reason.

Reply:
Fine with me, but you and me both know that "short-term" is more than often a lie we tell ourselves 😛

Reply:
The right answer here is "release branches" but that's a lot of work.